dev-requirements.txt

```diff
@@ -4,11 +4,7 @@
 # - can publish the built Helm chart (--publish)
 chartpress==0.2.1
 
-# yamllint is important for local development, CI
-# - used within tools/templates/lint-and-validate.py
+# yamllint and pytest are important for local development, CI
+pytest==3.7.1
```

Comment:
Do not pin these so tightly, it causes problems for people's environments and prevents our tests from catching compatibility issues. Let's use `>=` here, and pin to something like:

```
pytest>=3.3
requests>=2
```

Reply:
Ah okay!